articles/api/management/v2/user-search/v3/index.md

```diff
@@ -0,0 +1,304 @@
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "q",
+          "value": "logins_count:[100 TO 200]"
+        },
+        {
+          "name": "sort",
+          "value": "created_at:1"
+        },
+        {
+          "name": "search_engine",
+          "value": "v3"
+        }
+    ]
+}
+```
+
+::: note
+For more information on `sort` and other parameters, see the [Management API Explorer documentation](/api/v2#!/users/get_users).
+:::
+
+## Page Results
+
+::: warning
+Auth0 limits the number of users you can retrieve (1000). If the number of users you're retrieving exceeds this threshold, please redefine your search, use the [export job](/api/management/v2#!/Jobs/post_users_exports) or [contact support](https://support.auth0.com/) for further assistance.
+:::
+
+To page the user search results, use the `page`, `per_page`, and `include_totals` parameters when making your request:
+
+Parameter | Description
+----------|------------
+`page` | The page number, zero based.
+`per_page` | The amount of users per page.
+`include_totals` | Set to `true` to include a query summary as part of the result.
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "q",
+          "value": "logins_count:[100 TO 200]"
+        },
+        {
+          "name": "page",
+          "value": "2"
+        },
+        {
+          "name": "per_page",
+          "value": "10"
+        },
+        {
+          "name": "include_totals",
+          "value": "true"
+        },
+        {
+          "name": "search_engine",
+          "value": "v3"
+        }
+    ]
+}
+```
+
+::: note
+For more information on the `page`, `per_page` and other parameters, see the [Management API Explorer documentation](/api/v2#!/users/get_users).
+:::
+
+## Migrate from search engine v2 to v3
+
+The user search engine v2 will be deprecated soon, so we recommend migrating user search functionality to search engine v3 (`search_engine=v3`). Before you start migrating, there's a few things you should know:
+
+* Search values are case insensitive in `v3`.
+* `v3` limits the number of users you can retrieve (1000); if the number of users you're retrieving exceeds this threshold, please redefine your search, use the [export job](/api/management/v2#!/Jobs/post_users_exports) or [contact support](https://support.auth0.com/) for further assistance.
+* You can search for strings in `app_metadata`/`user_metadata` arrays, but not in nested `app_metadata`/`user_metadata` fields. See [searchable fields](#searchable-fields).
```

Comment:
we don't say it this though in that paragraph, do we? I can't find the reference where we sat that you cannot search in nested metadata fields.